components/brave_news/browser/combined_feed_parsing.cc

```diff
@@ -30,7 +30,8 @@ namespace {
 base::expected<mojom::FeedItemPtr, std::string> ParseFeedItem(
     const base::Value& value) {
   std::u16string error;
-  auto parsed_feed_item = api::combined_feed::Item::FromValue(value, &error);
+  auto parsed_feed_item =
+      api::combined_feed::Item::FromValueDeprecated(value, &error);
```

Comment:
why do we replace `FromValue->FromValueDeprecated`? looks like we use `auto` everywhere, so it should work with `optional<T>` version without changes.

Reply:
cc: @cdesouza-chromium ☝️

Reply:
I'm gonna have a look at this again, but the main issue is that FromValue got deprecated and at the same time the new version landed with the deprecation also required `base::Value::Dict`. A week later a different CL added an overload that also takes a `base::Value`. So the question is whether or not the second version, that takes `base::Value` has landed on `cr113` already, or is only on `cr114`.

Reply:
So, yes, it is better for us to wait for M114 to remove these deprecated calls.